includes/amp-helper-functions.php

```diff
@@ -325,42 +325,53 @@ function amp_correct_query_when_is_front_page( WP_Query $query ) {
  *
  * Themes can register support for this with `add_theme_support( AMP_Theme_Support::SLUG )`:
  *
- *      add_theme_support( AMP_Theme_Support::SLUG );
+ * ```php
+ * add_theme_support( AMP_Theme_Support::SLUG );
+ * ```
```

Comment:
Is use of triple backticks the correct way to embed code samples in phpdoc? Note in https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#description-formerly-long-description:

![image](https://user-images.githubusercontent.com/134745/103268241-0852c280-4968-11eb-815f-2da0192034ed.png)

But apparently inenting text is an alternative to using triple-backtick blocks. https://stackoverflow.com/a/40036952/93579

The WordPress coding standards may be outdated as well.

Reply:
When indentation is used, IntelliJ renders this as:

![image](https://user-images.githubusercontent.com/134745/103304535-b3da3200-49bd-11eb-9d92-678794c59c36.png)

When using backtick blocks, it is rendered as:

![image](https://user-images.githubusercontent.com/134745/103304610-eb48de80-49bd-11eb-9e22-a2c2107edec8.png)

So both work fine in that regard.

Reply:
This is for the tooltips, naturally.

I'm not seeing syntax highlighting for the backtick code blocks or indented code. But you are? Did you have to enable that somehow?

The backtick code blocks would be superior in this case since the language can be specified.

Reply:
> I'm not seeing syntax highlighting for the backtick code blocks or indented code. But you are? Did you have to enable that somehow?

Hmm yea that's weird. I have the same problem here as well. It renders just fine in Visual Studio Code, however:

![image](https://user-images.githubusercontent.com/16200219/103305616-844e1880-4a03-11eb-8223-d0c79f5f12fc.png)

Reply:
On second look this preview is only shown when you have a PHP extension enabled, such as "PHP Intelephense".